### PR TITLE
Tt mailer edits

### PR DESCRIPTION
**Pull request by ankitshah811**
Should *only* be merged after #453

### CODE REVIEW

**Comment by ankitshah811** [comment]
It's important to note that some of the emails in the attendance mailer spec are not getting sent out (or, at the least, weren't showing up on mailcatcher). This was the case before I made any changes, and **maybe** it's related to #331. What I noticed from my runthroughs:

* `attendance/registration` is not being sent
* `attendance/waitlist` is not being sent
* `reminder` (both of them) are being sent immediately upon tea time registration
   * they should be sent our 48 and 12 hours before a tea time, respectively
* I don't know about `attendance/waitlist_free_spot` (didn't observe it yet)
* I don't know about `tea_time/followup` emails (any of them)

**Comment by nickbarnwell** [comment]
Mails work differently in dev than production. Someone broke something if registration and waillist mails aren't being sent. go have fun with `git bisect` to find out who

**Comment by ankitshah811** [comment]
@nickbarnwell is there something I could read or general pointers that could help me understand what the diffs btwn dev behavior and production behavior are? Is this specifically relevant to both reminders being sent immediately?

**Comment by mehulkar** [comment]
Looks like `attendance.send_mail` is the method that sends attendance/registration email. I see it was replaced with `queue_reminders` in 778d412ed8fa63bbd0a2d26c256f5d7e8034e4de, but `queue_reminders` doesn't send that one.

**Comment by ankitshah811** [comment]
@mehulkar 

1. yes, reminder emails are going to come from "The Robots at Tea With Strangers" instead. a key piece of feedback we were getting from many hosts was that having emails they weren't composing being automatically sent to attendees was throwing them off and often led to confused communications when they would, at times, send custom reminder emails manually to send add'l details. I'll soon be filing an issue speccing customizable reminder emails for hosts, so the next step is to have the current email set up as the default/backup, but if a host customizes their reminders, they can override these emails that are sent by the robots. those emails would come from their email addresses
2. yup. I realized that confirmation emails, reminder emails, and waitlist emails were all much longer than they needed to be. and I want to test an assumption and observe the data comparing attendance before this switch and after this swtich — if we can shorten the reminder emails and make them more direct, they might be consumed more and lead to cancelations over no shows, which can lend itself to a stronger attendance

File: app/views/attendance_mailer/waitlist.markerb [hunk 1]
```diff
@@ -1,10 +1,10 @@
-Hey hey <%= @user.name %>!
-
-Just confirming that you have been added to the wait list for the following tea time:
+Just confirming that you've been added to the wait list for tea time with <%= @tea_time.host %>.
```
Comment (mehulkar) [inline app/views/attendance_mailer/waitlist.markerb]: this should be `@tea_time.host.name` probably

Comment (mehulkar) [inline app/views/attendance_mailer/waitlist.markerb]: i'll fix it

**Pull request merged**